README.md

```diff
@@ -147,6 +147,27 @@ Available [tags](https://docs.ansible.com/ansible/latest/user_guide/playbooks_ta
 
 - - -
 
+### Engine and Federated API applications
+
+Available playbooks to deploy both the Open Terms Archive Engine and Federated API applications on a single server.
+
+| Playbook name | Description | Command example |
+| --- | --- | --- |
+| `engine_and_federated_api.infrastructure` | Set up and configure the infrastructure required by the Open Terms Archive engine and federated API applications | `ansible-playbook opentermsarchive.deployment.engine_and_federated_api.infrastructure` |
+| `engine_and_federated_api.application` | Deploy the Open Terms Archive engine and federated API applications | `ansible-playbook opentermsarchive.deployment.engine_and_federated_api.application` |
+| `engine_and_federated_api.all` | Set up infrastructure and deploy the Open Terms Archive engine and federated API applications | `ansible-playbook opentermsarchive.deployment.engine_and_federated_api.all` |
+
+#### Configuration
+
+Available variables are listed below, along with default values:
+
+| Variable | Description | Default value | Required |
+| --- | --- | --- | --- |
+| `ota_reverse_proxy_engine_path` | Path where the API of the engine will be available | `/collection-api` | - |
```

Comment:
It's a bit confusing for me to say that the API of the engine will be available at `/collection-api` because the path does not bear the name of what it is claimed to be. 
```suggestion
| `ota_reverse_proxy_engine_path` | Path where the API of the collection will be available | `/collection-api` | - |
```
If that feedback seem to you a relevant one, it will have to be applied to your other changes so that we no longer speak of the engine API but the collection API.